lib/runtime/fetch.js

```diff
@@ -12,7 +12,9 @@ async function fetchShim (...args) {
 
   // In Cloudflare Workers, no upstream requests
   // get chunked
-  req.headers.set('transfer-encoding', '')
+  if (req.body) {
+    req.headers.set('transfer-encoding', '')
```

Comment:
As far as I can see, this line here is causing the concerns for me as mentioned in #33 

Would the following suggestion also work?
```js
    req.headers.delete('transfer-encoding')
```

Reply:
The intention with that line is to disable chunked transfer-encoding. When the body of a request is a Stream (node or a browser), node.js will automatically set the `transfer-encoding` to chunked. For the most part that was ok except that any request that gets cloned has a body of type `ReadableStream` even if the original body was created from string or something. This was causing problems for us here at DSC because some upstream services don't support chunked transfer-encoding. This was a solve (an admittedly hacky one at that) for this problem. I guess I'll have to reevaluate this approach. Now that we're using a forked version of node-fetch, I think the real solution will be to propagate the original body type across calls to `clone()`

Reply:
Let's move this discussion to #33 though